forta-core-monitoring/src/agent.config.json

```diff
@@ -1,7 +1,7 @@
 {
   "abi": [
     "event Transfer(address indexed from, address indexed to, uint256 indexed tokenId)",
-    "event StakeThresholdChanged(uint256 min, uint256 max, bool activated)",
+    "event StakeThresholdChanged(uint256 indexed chainId, uint256 min, uint256 max, bool activated)",
```

Comment:
on second look, there is a different StakeThresholdChanged event defined for agent registry contract vs scanner registry contract (only the scanner event has the chainId parameter). we probably want to monitor events for both agents and scanners